.cursor/commands/cmt.md

```
@@ -0,0 +1,168 @@
+
+**Examples:** `/cmd cmt`, `/cmd cmt noask`, `/cmd cmt stage all`, `/cmd cmt noask stage`
+
+## Valid Types
+
+**Types (REQUIRED - ALWAYS FIRST):** feat | fix | build | chore | ci | perf | refactor | revert | style | test
+
+## FORMAT - READ THIS CAREFULLY
+
+**CRITICAL: Type is MANDATORY and ALWAYS comes FIRST**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)